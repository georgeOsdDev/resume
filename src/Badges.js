import React from "react";
import Counter from "./Counter";

function Badges() {
  return (
    <p align="left">
      <a
        href="https://github.com/georgeOsdDev/resume/blob/main/LICENSE"
        style={{ marginRight: "2px" }}
      >
        <img
          src="https://img.shields.io/badge/license-0BSD-blue.svg"
          alt="Minimal Blog is released under the 0BSD license."
        />
      </a>
      <Counter />
    </p>
  );
}

export default Badges;
