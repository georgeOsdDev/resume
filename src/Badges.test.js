import React from 'react';
import renderer from 'react-test-renderer';
import Badges from './Badges';

test('snapshot', () => {
  const component = renderer.create(
    <Badges/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
