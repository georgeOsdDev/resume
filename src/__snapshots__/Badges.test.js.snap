// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<p
  align="left"
>
  <a
    href="https://github.com/georgeOsdDev/resume/blob/main/LICENSE"
    style={
      Object {
        "marginRight": "2px",
      }
    }
  >
    <img
      alt="Minimal Blog is released under the 0BSD license."
      src="https://img.shields.io/badge/license-0BSD-blue.svg"
    />
  </a>
  <img
    alt="Views: Loading..."
    src="data:image/svg+xml;utf8,%3Csvg%20width%3D%22107%22%20height%3D%2220%22%20viewBox%3D%220%200%201070%20200%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20role%3D%22img%22%20aria-label%3D%22Views%3A%20Loading...%22%3E%0A%20%20%3Ctitle%3EViews%3A%20Loading...%3C%2Ftitle%3E%0A%20%20%3ClinearGradient%20id%3D%22a%22%20x2%3D%220%22%20y2%3D%22100%25%22%3E%0A%20%20%20%20%3Cstop%20offset%3D%220%22%20stop-opacity%3D%22.1%22%20stop-color%3D%22%23EEE%22%2F%3E%0A%20%20%20%20%3Cstop%20offset%3D%221%22%20stop-opacity%3D%22.1%22%2F%3E%0A%20%20%3C%2FlinearGradient%3E%0A%20%20%3Cmask%20id%3D%22m%22%3E%3Crect%20width%3D%221070%22%20height%3D%22200%22%20rx%3D%2230%22%20fill%3D%22%23FFF%22%2F%3E%3C%2Fmask%3E%0A%20%20%3Cg%20mask%3D%22url(%23m)%22%3E%0A%20%20%20%20%3Crect%20width%3D%22418%22%20height%3D%22200%22%20fill%3D%22%23555%22%2F%3E%0A%20%20%20%20%3Crect%20width%3D%22652%22%20height%3D%22200%22%20fill%3D%22%2308C%22%20x%3D%22418%22%2F%3E%0A%20%20%20%20%3Crect%20width%3D%221070%22%20height%3D%22200%22%20fill%3D%22url(%23a)%22%2F%3E%0A%20%20%3C%2Fg%3E%0A%20%20%3Cg%20aria-hidden%3D%22true%22%20fill%3D%22%23fff%22%20text-anchor%3D%22start%22%20font-family%3D%22Verdana%2CDejaVu%20Sans%2Csans-serif%22%20font-size%3D%22110%22%3E%0A%20%20%20%20%3Ctext%20x%3D%2260%22%20y%3D%22148%22%20textLength%3D%22318%22%20fill%3D%22%23000%22%20opacity%3D%220.25%22%3EViews%3C%2Ftext%3E%0A%20%20%20%20%3Ctext%20x%3D%2250%22%20y%3D%22138%22%20textLength%3D%22318%22%3EViews%3C%2Ftext%3E%0A%20%20%20%20%3Ctext%20x%3D%22473%22%20y%3D%22148%22%20textLength%3D%22552%22%20fill%3D%22%23000%22%20opacity%3D%220.25%22%3ELoading...%3C%2Ftext%3E%0A%20%20%20%20%3Ctext%20x%3D%22463%22%20y%3D%22138%22%20textLength%3D%22552%22%3ELoading...%3C%2Ftext%3E%0A%20%20%3C%2Fg%3E%0A%20%20%0A%3C%2Fsvg%3E"
  />
</p>
`;
